lms/security.py

```diff
@@ -102,7 +102,7 @@ def get_policy(request: Request):  # noqa: C901, PLR0911
             # Routes that require the Google auth policy
             return LMSGoogleSecurityPolicy()
 
-        if path.startswith(("/api/dashboard")):
+        if path.startswith("/api/dashboard"):
```

Comment:
Here I reckon I was using the new style of passing multiple strings as a tuple (instead of doing multiple calls) but at some point removed items and ended up only with one.